lib/matplotlib/stackplot.py

```diff
@@ -34,37 +34,33 @@ def stackplot(axes, x, *args, **kwargs):
             stackplot(x, y)               # where y is MxN
             stackplot(x, y1, y2, y3, y4)  # where y1, y2, y3, y4, are all 1xNm
 
-    baseline : string
-        One of ``['zero', 'sym', 'wiggle', 'weighted_wiggle']``.
-
-        Method used to calculate the baseline. ``'zero'`` is just a
-        simple stacked plot.
-        ``'sym'`` is symmetric around zero and is sometimes called
-        'ThemeRiver'.
-        ``'wiggle'`` minimizes the sum of the squared slopes.
-        ``'weighted_wiggle'`` does the same but weights to account for size of
-        each layer. It is also called 'Streamgraph'-layout. More details
-        can be found at http://leebyron.com/streamgraph/.
+    baseline : ['zero' | 'sym' | 'wiggle' | 'weighted_wiggle']
```

Comment:
in the doc for y (just above): OR -> or (lowercase)

Reply:
changed.